libs/blocks/table/table.js

```diff
@@ -149,7 +149,7 @@ function handleAddOnContent(table) {
       el?.insertAdjacentElement(order === 'before' ? 'beforebegin' : 'afterend', tag);
     });
   });
-  setTimeout(() => handleEqualHeight(table, '.row-heading'), 0);
+  table.addEventListener('mas:resolved', debounce(() => { handleEqualHeight(table, '.row-heading'); }));
```

Comment:
Would this run for tables without any pricing in them? That use-case should also be considered and `mas:resolved` might not be dispatched in that case. It might be that both the initial statement and the new one need to co-exist.